e2e/only-option/cli-options.txt

```diff
@@ -0,0 +1 @@
+--only="Rector\DeadCode\Rector\ClassMethod\RemoveUnusedPrivateMethodRector"
```

Comment:
Could you also add single quotes and double slashes mix?
That's what caused troubles in past.

Reply:
that should be equal https://3v4l.org/u19fl, but indeed needs verification as cli value maybe transform different

Reply:
Still, these tests should be added so we have verifiction in CI.

Reply:
Also cases without `=` should be covered, as they behave differently in different OSs:

```bash
--only "Rector\DeadCode\Rector\ClassMethod\RemoveUnusedPrivateMethodRector"
--only 'Rector\DeadCode\Rector\ClassMethod\RemoveUnusedPrivateMethodRector'
```

Reply:
I've added e2e-Tests for
- Single quotes with =
- Single quotes with double backslashes with =
- Single quotes without =
- Double quotes without =